bin/mail/2016-09-27-latam-hoc-minecraft/generate-mailing-lists.rb

```diff
@@ -0,0 +1 @@
+../2016-09-15-latam-hoc-starwars/generate-mailing-lists.rb
```

Comment:
Presumably, this will replace (overwrite) the existing `latam-contacts.csv` file?